ext/symengine/ruby_utils.h

```diff
@@ -0,0 +1,9 @@
+#ifndef RUBY_UTILS_H_
```

Comment:
I was thinking to keep the functions for C in symengine_utils.h, while having the functions exposed in Ruby in ruby_utils.h (as per the convention of other .h files)...

What do you think?